nox/command.py

```diff
@@ -21,11 +21,16 @@
 from nox.logger import logger
 from nox.popen import popen
 
+if sys.version_info < (3, 8):
+    from typing_extensions import Literal
+else:
+    from typing import Literal
+
```

Comment:
Unfortunately, this will need `# pragma: no cover` on both branches, since we don't combine coverage data from the various Python sessions.